packages/data-service/test/helpers.ts

```diff
@@ -92,5 +95,19 @@ export function createMongoClientMock({
     },
   };
 
-  return client as unknown as MongoClient;
+  const connectionString = hosts.reduce<string>(
+    (connectionString, { host, port }, index) => {
+      const hostWithPort = port ? `${host}:${port}` : host;
+      if (index === 0) {
+        return `${connectionString}://${hostWithPort}`;
+      }
+      return `${connectionString},${hostWithPort}`;
+    },
+    'mongodb'
+  );
```

Comment:
Do you just want to use the `ConnectionString` class from the `mongodb-connection-string-url` package here? Here, you'd technically have to worry about things like escaping characters in `host` and `port` as well (e.g. for IPv6 hosts)

Reply:
yea that is right. I will make the adjustments.